src/dotnet-install.ps1

```diff
@@ -215,6 +226,54 @@ function Get-CLIArchitecture-From-Architecture([string]$Architecture) {
     }
 }
 
+
+function Get-NormalizedQuality([string]$Quality) {
+    Say-Invocation $MyInvocation
+
+    if ([string]::IsNullOrEmpty($Quality)) {
+        return ""
+    }
+
+    switch ($Quality.ToLower()) {
+        { ($_ -eq "daily") -or ($_ -eq "signed") -or ($_ -eq "validated") -or ($_ -eq "preview")} { return $Quality.ToLower() }
```

Comment:
`-eq` is case insensitive. the toLower is unnecessary.

I would use `@("daily", "signed", ...) -contains $_` instead of ORing the checks.